### PR TITLE
Introduce p.Info and p.Output to log messages and command output (respectively)

### PR DESCRIPTION
**Pull request by DiogoFerrao**

### CODE REVIEW

**Review by vicentepinto98 — COMMENTED** [review]
This PR is missing the replacement of `cmd.OutOrStdout()` with `p.OutOrStdout()` inside the method `Display` on the `pager ` pkg. 

This will complete the replacement of the uasge of `cmd` output methods with `print` methods

File: internal/cmd/auth/login/login.go [hunk 1]
```diff
@@ -27,7 +28,7 @@ func NewCmd() *cobra.Command {
 				return fmt.Errorf("authorization failed: %w", err)
 			}
 
-			cmd.Println("Successfully logged into STACKIT CLI.")
+			p.Info("Successfully logged into STACKIT CLI.\n")
```
Comment (vicentepinto98) [inline internal/cmd/auth/login/login.go:31]: :+1: We could consider a `p.Infoln` method but I think it's cleaner like this

File: internal/cmd/config/set/set.go [hunk 2]
```diff
@@ -70,7 +71,7 @@ func NewCmd() *cobra.Command {
 			}
 
 			if model.SessionTimeLimit != nil {
-				cmd.Println("Authenticate again to apply changes to session time limit")
+				p.Info("Authenticate again to apply changes to session time limit")
```
Comment (vicentepinto98) [inline internal/cmd/config/set/set.go:74]: This is rather a warning, you can leave it for the PR to replace the warnings to be more clear

**Comment by DiogoFerrao** [comment]
> This PR is missing the replacement of `cmd.OutOrStdout()` with `p.OutOrStdout()` inside the method `Display` on the `pager ` pkg.
> 
> This will complete the replacement of the uasge of `cmd` output methods with `print` methods

I was considering doing that on a separate PR, but I can also add it to this one

**Comment by vicentepinto98** [comment]
> This PR is missing the replacement of cmd.OutOrStdout() with p.OutOrStdout() inside the method Display on the pager  pkg.

Will you do this in a follow-up PR?

**Review by vicentepinto98 — APPROVED** [review]

**Comment by DiogoFerrao** [comment]
> > This PR is missing the replacement of cmd.OutOrStdout() with p.OutOrStdout() inside the method Display on the pager  pkg.
> 
> Will you do this in a follow-up PR?

Yes, I didn't want to further complicate this PR

**Pull request merged**